skabase/SKABaseDevice/SKABaseDevice/SKABaseDevice.py

```diff
@@ -416,17 +416,17 @@ def write_testMode(self, value):
     dtype_out='str', 
     )
     @DebugIt()
-    def GetMetrics(self):
-        # PROTECTED REGION ID(SKABaseDevice.GetMetrics) ENABLED START #
-        ### TBD - read the value of each of the attributes in the MetricList
-        with exception_manager(self):
-            args_dict = {'with_value': True, 'with_commands': False,
-                         'with_metrics': True, 'with_attributes': False}
-            device_dict = self._get_device_json(args_dict)
-            argout = json.dumps(device_dict)
-
-        return argout
-        # PROTECTED REGION END #    //  SKABaseDevice.GetMetrics
+    # def GetMetrics(self):
```

Comment:
As mentioned in previous PR, please remove unused code completely rather than just commenting it out.
Same comment applies to many of the files in the PR.

Reply:
Removed. Commit f51ec80 addresses this.